insecure_generate_tls.sh

```diff
@@ -95,4 +104,5 @@ openssl dhparam -in "${CAFILE}" -out "${DHFILE}" 2048
 
 echo "Certificate chain is at: ${CHAINFILE}"
 echo "Private key is at: ${KEYFILE}"
-
+echo ""
+echo "**Remember** the default action is to store the files in /tmp/ so they'll be deleted on reboot! Set the KANI_TMP environment variable before running this script if you want to change that. You'll need to update server config elsewhere if you do, however."
```

Comment:
Believe it or not on macos, if it hibernates it'll also clear this folder.

Reply:
That's what you get for leaving your machine alone for so long. 😭